arangod/Graph/PathManagement/PathValidatorOptions.h

```diff
@@ -128,7 +129,6 @@ class PathValidatorOptions {
   aql::FixedVarExpressionContext& getExpressionContext();
 
   // @brief If a graph is asked for the first vertex and that is filtered
-  // it can be removed for 3.9 => nextVersion.
```

Comment:
This code comment turned out not to be true: SmartGraphEngine.cpp can call this function.